lib/notify.js

```diff
@@ -50,11 +51,18 @@ module.exports = CoreObject.extend({
           var errorMessage = serviceKey + ' => ' + error;
 
           plugin.log(errorMessage, { color: 'red' });
+
+          if (critical === true) {
+            return Promise.reject(error);
+          }
         });
     } else {
       var warningMessage = 'No request issued! Request options invalid! You have to specify `url`, `headers`, `method` and `body`.';
       plugin.log(serviceKey+' => '+warningMessage, { color: 'yellow', verbose: true });
```

Comment:
Small nitpick but I think it would be more consistent if the logging color would be red when we set the notification to critical.